backend/test/config.spec.js

```diff
@@ -74,7 +74,8 @@ describe('config', function () {
         OIDC_ISSUER: 'issuer',
         OIDC_CLIENT_ID: 'client_id',
         OIDC_CLIENT_SECRET: 'client_secret', // pragma: whitelist secret
-        OIDC_REDIRECT_URI: 'redirect_uri'
+        OIDC_REDIRECT_URI: 'redirect_uri',
+        OIDC_CA: 'my_ca'
```

Comment:
```suggestion
        OIDC_REDIRECT_URI: 'redirect_uri'
```
OIDC_CA is not a required environment variable!